lib/resin-lint.ts

```diff
@@ -1,4 +1,4 @@
-import * as Promise from 'bluebird';
+import * as BluePromise from 'bluebird';
```

Comment:
We generally do `import * as Bluebird from 'bluebird';` and you've also done this in mocha-test-lint.ts, so it'd be better to stay consistent.